esp/services/ws_access/ws_accessService.cpp

```diff
@@ -176,6 +177,14 @@ void Cws_accessEx::init(IPropertyTree *cfg, const char *process, const char *ser
         m_rawbasedns.append(*onedn.getLink());
     }
 
+    xpath.setf("Software/EspProcess[@name=\"%s\"]/@PageCacheTimeoutSeconds", process);
+    unsigned pageCacheTimeoutSeconds = cfg->getPropInt(xpath.str(), 0); //0: use default.
+    if (pageCacheTimeoutSeconds > 0)
+        setPageCacheTimeoutMilliSeconds(pageCacheTimeoutSeconds);
+    xpath.setf("Software/EspProcess[@name=\"%s\"]/@MaxPageCacheItems", process);
+    unsigned maxPageCacheItems = cfg->getPropInt(xpath.str(), 0); //0: no limit.
```

Comment:
use hasProp and pass value through to setMaxPageCacheItems.
I think 0 should mean cache off.